src/static/riot/competitions/detail/_tabs.tag

```diff
@@ -279,6 +283,119 @@
         </div>
     </div>
 
+    <script>
```

Comment:
Kind of misleading. This was just re-ordering the tag to be 
```html
<template/>
<script/>
<style/>
```
instead of 
```html
<template/>
<style/>
<script/>
```